backend/src/main/kotlin/org/loculus/backend/service/submission/SubmissionDatabaseService.kt

```diff
@@ -481,7 +486,7 @@ open class SubmissionDatabaseService(
         }
 
         val scopeCondition = if (scope == ApproveDataScope.WITHOUT_WARNINGS) {
-            not(SequenceEntriesView.entriesWithWarnings)
+            not(SequenceEntriesView.hasWarnings)
```

Comment:
Would it make sense to rename the variable to `warnignCondition`?

Reply:
I have now made this particular condition private and refactored that whole bit to maybe make it clearer what's going on.

https://github.com/loculus-project/loculus/pull/3125/commits/59f73282057b80b8a297cb4a314f94b15129c208